### DISCUSSION TITLE
module 'lv' has no member 'tabview'

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#21082] [Q&A]
i started with a fresh auto-configed LVG Display. i started with the example and was able to add a sliter above the three buttons.
But i can't create a tabvie object.

t.be:
```
...
tab = lv.tabview(scr)
...
```

berry output:
```
load("t.be")
BRY: failed to run compiled code 'attribute_error' - module 'lv' has no member 'tabview'
```

**Comment by sfromis** [comment]
Checking `my_user_config.h` reveals that this is an optional build time feature, to be enabled if you need it.
`// #define BE_LV_WIDGET_TABVIEW`
